### PR TITLE
Fix/Disable send text area and show loader

### PR DESCRIPTION
**Pull request by 0xavro**
Disable send message text area and show loader when message is sending or waiting approval

### CODE REVIEW

File: packages/dialect-react-ui/components/Chat/screens/Main/pages/ThreadPage/MessageInput.tsx [hunk 1]
```diff
@@ -41,14 +44,21 @@ export default function MessageInput({
...
               <ButtonBase
                 className="dt-button dt-absolute dt-inset-y-0 dt--right-2 dt-flex dt-items-center dt-pr-3 disabled:dt-cursor-not-allowed"
-                disabled={disabled}
+                disabled={disableSendButton}
               >
-                <icons.arrowsmright
-                  className={clsx(sendButton, disabled ? 'dt-opacity-50' : '')}
-                />
+                {
+                  disableTextarea ? (
```
Comment (cbosborn) [inline packages/dialect-react-ui/components/Chat/screens/Main/pages/ThreadPage/MessageInput.tsx:54]: this name is a bit misleading here but thinking on it I'm not sure what's a better set of `bool`s. most explicit set would be

- `buttonDisabled`
- `inputDisabled`
- `sending`
- `creating`

maybe overkill though

**Review by cbosborn — CHANGES_REQUESTED** [review]
@0xavro lgtm, nice job. one minor comment but up to you if you wanna address or just merge. let's target `dev` though, and add a note in the `CHANGELOG` unreleased.

**Comment by cbosborn** [comment]
(requested change is just targeting dev & adding changelog)

File: packages/dialect-react-ui/components/Chat/screens/Main/pages/ThreadPage/MessageInput.tsx (hunk 1 above)
Comment (0xavro) [reply packages/dialect-react-ui/components/Chat/screens/Main/pages/ThreadPage/MessageInput.tsx:54]: I'll go with `inputDisabled`

**Review by cbosborn — APPROVED** [review]

**Pull request merged**